src/Components/PostCard.js

```diff
@@ -13,11 +13,11 @@ export default function Posts() {
 			<p>{data[0].contact}</p>
 
 			<address>
-				<CardAdresse>{data[0].mail}</CardAdresse>
+				<CardAddress>{data[0].mail}</CardAddress>
 			</address>
 
 			<address>
-				<CardAdresse>{data[0].mobile}</CardAdresse>
+				<CardAddress>{data[0].mobile}</CardAddress>
 			</address>
```

Comment:
I am not sure what your are intending to do in future. You could rethink for the next branch just to use one address-tag.

Nice work 
approved